Modules/_heapqmodule.c

```diff
@@ -694,13 +693,29 @@ Believe me, real good tape sorts were quite spectacular to watch!\n\
 From all times, sorting has always been a Great Art! :-)\n");
 
 
+static int
+heapq_exec(PyObject *m)
+{
+    PyObject *about = PyUnicode_DecodeUTF8(__about__, strlen(__about__), NULL);
```

Comment:
PyUnicode_FromString() is shorter and uses UTF-8 ;-)

```suggestion
    PyObject *about = PyUnicode_FromString(__about__);
```